File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-persons-vetting_item.ts


import { IGetPersonsVetting_Item } from './iget-persons-vetting_item';

export class GetPersonsVetting_Item implements IGetPersonsVetting_Item {
  
	public PersonsVettingID: number = 0;
	public PersonID: number = 0;
	public VettingPersonStatusID: number = 0;
	public VettingStatus: string = "";
	public BatchID: string = "";
	public TrackingNumber: string = "";
	public VettingValidStartDate?: Date;
	public VettingValidEndDate?: Date;
	public RemovedFromVetting?: boolean;
  
}

File: GTTS/INL.VettingService.Models/GetPersonsVetting_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class GetPersonsVetting_Item : IGetPersonsVetting_Item
    {
        public long PersonsVettingID { get; set; }
        public long PersonID { get; set; }
        public int VettingPersonStatusID { get; set; }
        public string VettingStatus { get; set; }
        public string BatchID { get; set; }
        public string TrackingNumber { get; set; }
        public DateTime? VettingValidStartDate { get; set; }
        public DateTime? VettingValidEndDate { get; set; }
        public bool? RemovedFromVetting { get; set; }
    }
}
